cmd/install/addon/cmd.go

```diff
@@ -261,6 +269,12 @@ func run(cmd *cobra.Command, argv []string) {
 						os.Exit(1)
 					}
 				}
+
+				if len(input.Options) > 0 && !contains(input.Options, val) {
```

Comment:
We can use the `Contains` function from `pkg/helper/helpers.go`
```suggestion
				if len(input.Options) > 0 && !helper.Contains(input.Options, val) {
```